src/Features/Core/Portable/SimplifyInterpolation/Helpers.cs

```diff
@@ -74,9 +74,13 @@ private static IOperation Unwrap(IOperation expression)
                     unwrapped = invocation.Instance;
                     formatString = format;
 
-                    unnecessarySpans.AddRange(invocation.Syntax.Span
-                        .Subtract(invocation.Instance.Syntax.FullSpan)
-                        .Subtract(GetSpanWithinLiteralQuotes(virtualCharService, argumentValue.Syntax)));
+                    var subtractedSpans = invocation.Syntax.Span.Subtract(invocation.Instance.Syntax.FullSpan);
+                    if (TryGetSpanWithinLiteralQuotes(virtualCharService, argumentValue.Syntax) is TextSpan stringSpan)
+                    {
+                        subtractedSpans = subtractedSpans.Subtract(stringSpan);
+                    }
+
+                    unnecessarySpans.AddRange(subtractedSpans);
```

Comment:
You don't like leveraging the fact that subtracting an empty span is a no-op?

Reply:
it was an intermediary step.